bin/thinkingThingsClient

```diff
@@ -214,6 +261,11 @@ var commands = {
         description: '\tSend a new gps measure',
         handler: gpsLocation
     },
+    'gsm': {
+        parameters: ['mcc', 'mnc', 'cell-id', 'lac', 'dbm'],
+        description: '\tSend a new gps measure',
```

Comment:
I noticed this feature refers to GSM positioning but in this description it is mentioned `gps measure` :) Just to avoid confusion about the geo-positioning mechanism or technology being used... ;) Maybe something such as 'Send new GSM positioning data/measure' :) No need to say that I leave it to your discretion... :)